docs/sdk_example.ipynb

```diff
@@ -65,8 +65,8 @@
    "source": [
     "## Authenticating to the API\n",
     "\n",
-    "The SDK will attempt to pull your [NASA Earthdata Login](https://urs.earthdata.nasa.gov/) credentials out of `~/.netrc`\n",
-    "by default, or you can pass your credentials in directly"
+    "There are a variety of strategies to authenticate the HyP3 API in the SDK. If you want to explore all options for authentication, [check out our HyP3 Authentication notebook](https://github.com/ASFHyP3/hyp3-sdk/blob/main/docs/hyp3_authentication.ipynb). \n",
```

Comment:
```suggestion
    "There are several approaches available for authenticating the HyP3 API in the SDK. Refer to our [HyP3 Authentication notebook](https://github.com/ASFHyP3/hyp3-sdk/blob/main/docs/hyp3_authentication.ipynb) for more information and sample code for the different options. \n",
```

Reply:
Do we want to at least mention the different approaches, even if we don't go into detail? I waffled on that. I think @jtherrmann was advocating for not adding any specifics here and keeping it simple for future changes, but I wonder if it might be helpful.